src/sgx/libpoet_bridge/CMakeLists.txt.bak

```diff
@@ -0,0 +1,95 @@
+# Copyright 2018 Intel Corporation
```

Comment:
Is this .bak file here by accident? If it isn't required, I'd suggest removing it.

Reply:
Yes Amol, this is committed by accident and will be removed in the upcoming PR.